CHANGELOG.md

```diff
@@ -26,6 +26,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   `ui.editor` config. There is also a new `$JJ_EDITOR` environment variable,
   which has even higher priority than the config.
 
+* The new revset function `connected(x)` is the same as `x:x`.
+
+* The new revset function `roots(x)` finds commits in the set that don't have
```

Comment:
Isn't `roots` the function that finds commits without ancestors?

Reply:
Good catch! I'll copy over the description from the .md file. Thanks!